langchain/vectorstores/milvus.py

```diff
@@ -208,6 +208,13 @@ def _init(
         self._create_search_params()
         self._load()
 
+    def _list_all_collections(self) -> None:
+        """Return all the available collections"""
+        from pymilvus import utility
+        
+        collections = utility.list_collections(using=self.alias)
+        return collections #TODO: Added this new function
```

Comment:
could you clarify the todo

Reply:
oh, it was just a note to show where the change was, that was it. Nothing needs to be done, you can delete it.

It's my first pull request and I didn't realise they highlighted the whole block lol.